hnn_core/gui.py

```diff
@@ -0,0 +1,1028 @@
+"""IPywidgets GUI."""
+
+# Authors: Mainak Jas <mjas@mgh.harvard.edu>
+#          Huzi Cheng <hzcheng15@icloud.com>
+
+import codecs
+import multiprocessing
+import os.path as op
+import numpy as np
+import os
+import matplotlib.pyplot as plt
+from IPython.display import display
+from ipywidgets import (HTML, Accordion, AppLayout, BoundedFloatText, Button,
+                        Dropdown, FileUpload, FloatLogSlider, FloatText, HBox,
+                        IntText, Layout, Output, RadioButtons, Tab, Text, VBox,
+                        interactive_output, GridspecLayout)
```

Comment:
first built in imports. Then, other libraries, then more specific libraries

```suggestion
import os
import os.path as op
import codecs
import multiprocessing

import numpy as np
import matplotlib.pyplot as plt

from IPython.display import display
from ipywidgets import (HTML, Accordion, AppLayout, BoundedFloatText, Button,
                        Dropdown, FileUpload, FloatLogSlider, FloatText, HBox,
                        IntText, Layout, Output, RadioButtons, Tab, Text, VBox,
                        interactive_output, GridspecLayout)
```

Reply:
using isort gives a slightly different, alphabetically sorted version, is this ok?
```Python
import codecs
import multiprocessing
import os
import os.path as op

import matplotlib.pyplot as plt
import numpy as np
from IPython.display import display
from ipywidgets import (HTML, Accordion, AppLayout, BoundedFloatText, Button,
                        Dropdown, FileUpload, FloatLogSlider, FloatText,
                        GridspecLayout, HBox, IntText, Layout, Output,
                        RadioButtons, Tab, Text, VBox, interactive_output)

import hnn_core
from hnn_core import (JoblibBackend, MPIBackend, jones_2009_model, read_params,
                      simulate_dipole)
from hnn_core.params import (_extract_drive_specs_from_hnn_params, _read_json,
                             _read_legacy_params)
from hnn_core.viz import plot_dipole
```